src/css/_section-intro.css

```diff
@@ -3,3 +3,10 @@
  *
  * Start all rules with .section-intro
  */
+.section-intro .section-heading{
+  font-family: 'Revalia', cursive  ;
```

Comment:
hey Anthony, this is just to show you how to leave a comment on a PR. Can you use Times News Font